packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -977,6 +977,12 @@ export interface IRetriableFailureResult {
     readonly retryAfterSeconds?: number;
 }
 
+// @alpha
```

Comment:
Too bad all these types are `@alpha`, it's weird.  I guess because someone could implement their own DataStore or DDS or something so all summarizer stuff is exposed?  Might be worth opening a feature for, to review and figure out how to hide most of this stuff, and even convert what is necessary to public.

Reply:
A lot of the summary results are alpha which makes all of these types alpha too. I am not sure why. Maybe because of ondemand summaries? Or maybe the summarize function on container runtime is alpha?